boards/arm/cc3220sf_launchxl/doc/cc3220sf_launchxl.rst

```diff
@@ -132,8 +136,7 @@ Prerequisites:
 
 #. Ensure CONFIG_XIP=y is set.
 
-   The default board configuration file can be found at
-   :file:`boards/arm/cc3220sf_launchxl/cc3220sf_launchxl_defconfig`.
+   Add a 'CONFIG_XIP=y' line to the project's prj.conf file.
```

Comment:
While this is OK, it would read better to just replace the "Ensure CONFIG_XIP=y is set" step with what you added